README.md

```diff
@@ -5,16 +5,23 @@ Application for demonstrating surface blending with ros
 
 Godel: Austrian logician and mathematician http://en.wikipedia.org/wiki/Kurt_G%C3%B6del
 
+### 3rd Party Dependencies
+==============
+- libqd-dev
```

Comment:
I've already submitted `rosdep` rules for `libqd-dev` in ros/rosdistro#6599.

Technically your right (it is a third-party dep), but a `rosdep check etc etc` should automatically install this for users (and binary builds (debs) will have it listed as a dependency).

Perhaps we should reword this a bit (as it now looks like this is something users would have to install themselves.